scss/_tables.scss

```diff
@@ -81,6 +81,11 @@
   tbody tr {
     @include hover {
       background-color: $table-bg-hover;
+
+      > td,
+      > th {
```

Comment:
Yes indeed.
There was also children selector in [_table-row.scss](https://github.com/twbs/bootstrap/pull/21676/files/54123bbb734cd44cdaa152c013c12605e98d6442#diff-d25719387baa3cdfd3780f0b21e62b85) that were there before my PR. I removed them as well for consistency.
Good catch, thanks !